crates/swc_atoms/Cargo.toml

```diff
@@ -7,18 +7,25 @@ edition       = "2021"
 license       = "Apache-2.0"
 name          = "swc_atoms"
 repository    = "https://github.com/swc-project/swc.git"
-version       = "0.4.9"
+version       = "0.4.10"
 
 [lib]
 bench = false
 
 [features]
-rkyv-impl = ["rkyv", "bytecheck"]
+__rkyv = []
+# Enably rkyv serialization with stable version of rkyv.
+rkyv-impl = ["__rkyv", "rkyv-stable"]
+# Enable rkyv serialization with updated version of rkyv, along with bytecheck.
+rkyv-bytecheck-impl = ["__rkyv", "rkyv-latest"]
 
 [dependencies]
-bytecheck    = { version = "0.6.9", optional = true }
-once_cell    = "1"
-rkyv         = { version = "=0.7.37", optional = true }
+bytecheck   = { version = "0.6.9", optional = true }
+once_cell   = "1"
+rkyv-stable = { package = "rkyv", version = "=0.7.37", optional = true }
+# This is to avoid cargo version selection conflict between rkyv=0.7.37 and other versions, as it is strictly pinned
+# cannot be merged.
+rkyv-latest  = { git = "https://github.com/rkyv/rkyv", package = "rkyv", tag = "v0.7.38", optional = true }
```

Comment:
I'm not sure if cargo publish will work with this.
Did you verify it?

Reply:
ugh right, it needs version.

Reply:
ok, this is tricky:


```
error: failed to verify package tarball

Caused by:
  failed to select a version for `rkyv`.
      ... required by package `swc_atoms v0.4.10 (/Users/ojkwon/github/swc/target/package/swc_atoms-0.4.10)`
  versions that meet the requirements `=0.7.37` are: 0.7.37

  all possible versions conflict with previously selected packages.

    previously selected package `rkyv v0.7.38`
      ... which satisfies dependency `rkyv-latest = "=0.7.38"` of package `swc_atoms v0.4.10 (/Users/ojkwon/github/swc/target/package/swc_atoms-0.4.10)`

  failed to select a version for `rkyv` which could resolve this conflict
```

fails when try to verify publish. I expect this'll occur same on the user (swc_core) as well. :/

Reply:
Peeking bit, there's no way to escape (https://users.rust-lang.org/t/impact-of-pinning-dependencies/17634/16) with cargo. Probably as a shortcut, the quickest way is to publish dummy pkg doesn't have same pkg name instead of trying to alias. (Or branching, but that'll be more cumbersome for now)

Reply:
I ended up using forked version to have different pkg name for the versions we want to test (0.7.38, 39).